db/fixtures/ae_datastore/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb

```diff
@@ -13,9 +13,14 @@ def cloud?(prov_type)
 end
 
 def calculate_requested(options_hash = {})
-  {:memory  => get_total_requested(options_hash, :vm_memory) * 1024**2,
+  storage = if %w(google).include?(@miq_request.source.vendor)
```

Comment:
@billfitzgerald0120 You are just checking for a single provider.  No reason for the complexity.
`storage = if @miq_request.source.vendor == 'google'`